src/main/java/edu/ucla/library/services/utility/PropsWriter.java

```diff
@@ -0,0 +1,165 @@
+
+package edu.ucla.library.services.utility;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.util.Properties;
+
+public final class PropsWriter {
+
+    /**
+     * Properties file used as source for output file.
+    */
+    private static final Properties PROPS = new Properties();
+
+    /**
+     * Expected number of parameters passed to class.
+    */
+    private static final int ARGS_COUNT = 3;
+
+    /**
+     * Delimiter character for value keys in template file.
+    */
+    private static final String DELIMITER = "$";
+
+    /**
+     * Private constructor for PropsWriter class.
+    */
+    private PropsWriter() {
+        super();
+    }
+
+    /**
+     * Main method for command-line execution.
+     *
+     * @param args array of parameters
+    */
+    @SuppressWarnings("uncommentedmain")
+    public static void main(final String[] args) {
+        verifyArgs(args);
```

Comment:
Not suggesting you change this, but just thought you might be interested in this CLI parsing library: https://rvesse.github.io/airline/ -- it gives you file checks, etc., out of the box